src/frontend/js/modules/user/users-autocomplete-input/users-autocomplete-input.directive.js

```diff
@@ -1,13 +1,13 @@
 'use strict';
 
+const _ = require('lodash');
```

Comment:
1.  Even we do not have eslint now, but it is better if we keep our convention by adding 1 blank line after defining vars
2.  The commit message `Replace lodash Import ` is not good IMO.

Reply:
Fixed it